aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/table.v2_exp

```diff
@@ -52,7 +52,7 @@ Events:
 }
 {
     type:    0x1::transaction_fee::FeeStatement
-    data:    "9c0000000000000006000000000000000400000000000000c83a0200000000000000000000000000"
```

Comment:
These changes are unrelated to this PR, but were obtained by running `move_pr.sh -i2`.